frontend/src/scenes/funnels/FunnelViz.tsx

```diff
@@ -64,6 +68,13 @@ export function FunnelViz({
         }
     }
 
+    function buildChart(): void {
+        // Build chart only for the selected display view
+        if (!featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] || filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+            return buildFlowChart()
+        }
```

Comment:
Why can't this guard just go inside `buildChart` as it was above?

Reply:
There was a case-switch in this block when `FUNNEL_BAR_VIZ` was a totally separate chart type — this is left over cruft and will be removed.